test/localhost.crt

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
If everyone's going to be making their own copy of these they probably shouldn't be tracked, right?